aiida_mlip/calculations/train.py

```diff
@@ -158,11 +186,16 @@ def prepare_for_submission(
         # Initialize cmdline_params with train command
         codeinfo.cmdline_params = ["train"]
         # Create the rest of the command line
-        cmd_line = {}
-        cmd_line["mlip-config"] = config_copy
+        cmd_line = {"mlip-config": config_copy}
+        if self.inputs.fine_tune:
+            cmd_line["fine-tune"] = None
+
         # Add cmd line params to codeinfo
         for flag, value in cmd_line.items():
-            codeinfo.cmdline_params += [f"--{flag}", str(value)]
+            if value is None:
```

Comment:
Might we ever want to actually pass None as a value?

Probably not since it'd end up as a "None" string, so probably just something to be aware of...

Reply:
fine_tune can be True or False, and only if it's true we add {fine-tune:None} to the cmdline dict so that here it just adds `--fine-tune` to the command line  instead of writing like `janus --fine-tune True` it will just write `janus --fine-tune`)